src/Hammer.js

```diff
@@ -91,34 +91,33 @@ function updateHammer (hammer, props) {
 	});
 }
 
-var HammerComponent = React.createClass({
+class HammerComponent extends React.Component {
+    static displayName = 'Hammer';
 
-	displayName: 'Hammer',
-
-	propTypes: {
+    static propTypes = {
```

Comment:
Setting Statics will not work with Webpack if someone doesn't compile the node_modules folder. For the sake of compatibility you need to add PropTypes and displayName after instantiating the class:

```
HammerComponent.displayName = 'Hammer'
HammerComponent.propTypes = {
	className: PropTypes.string,
}
```